faculty/clients/experiment.py

```diff
@@ -416,6 +422,12 @@ def create_run(
         Returns
         -------
         ExperimentRun
+
+        Raises
+        ------
+        ExperimentNotActiveConflict
+            When the run that is being updated refers to an experiment that is
+            deleted or does not exist
```

Comment:
I would have thought we would return a 404 from the API in the case the experiment does not exist?

Reply:
We do. I am not quite sure I understand how we deal with the NOT_FOUND cases (which are also not caught in other endpoints). It seems to me like we just return the default `NotFound` exception from the `base` client. Do you think this is not correct?